typescript/submissions.d.ts

```diff
@@ -33,13 +33,17 @@ export interface NewFormsAppDraft {
    */
   previousFormSubmissionApprovalId?: string
   /** The id of the scheduled task that was started when the draft was saved. */
-  taskId?: number
+  taskId?: string
   /** The name of the scheduled task that was started when the draft was saved. */
   taskName?: string
   /** The id of the scheduled task group instance that the taskId is related to. */
   taskGroupInstanceId?: string
   /** The label of the scheduled task group instance that the taskId is related to. */
   taskGroupInstanceLabel?: string
+  /** The id of the scheduled task action that was used to complete the task. */
+  taskActionId?: string
+  /** The label of the scheduled task action that was used to complete the task. */
+  taskActionLabel?: string
```

Comment:
What do we need this one for?

Reply:
to display the label on the draft if they are offline (I also hate this)